docs/examples/lambda-aws/index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use MongoDB\Client;
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('The MONGODB_URI environment variable is not set');
+$client = new Client($uri);
+$planets = $client
+    ->selectDatabase('sample_guides')
+    ->selectCollection('planets')
```

Comment:
There are two possible alternatives here:
```suggestion
$planets = $client
    ->selectCollection('sample_guides', 'planets')
```

Alternatively, we can also rely on magic getters to achieve the same effect:

```suggestion
$planets = $client->sample_guides->planets
```